tests/e2e/fio_test.go

```diff
@@ -52,8 +52,10 @@ var _ = Describe("end to end test", func() {
 	}
 
 	Describe("creating fio job from multiple CRs", func() {
-		for i, fioCrDir := range fioCrDirs {
-			crName := fmt.Sprintf("fio-sample-%d", i)
+		for _, fioCrDir := range fioCrDirs {
+			splits := strings.Split(fioCrDir, "/")
```

Comment:
Minor: filepath has a splitter for paths: https://golang.org/pkg/path/filepath/#SplitList

Reply:
This splits by ":"s, I don't see why we'd need it here.

Reply:
sorry - this clearly not feasible here.